CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `Robot` now inherit from `compas.data.Data`
 * `RobotSemantics` now inherit from `compas.data.Data`
 * `Tool` now inherit from `compas.data.Data`
+* Fixed a number of Sphinx warnings
```

Comment:
You could skip changelog on this one, changelog should list entries that are relevant for a user of the library to understand what has changed, for developers we can check github log

Reply:
If you skip the changelog entry, you need to add a label to the PR so that the continuous integration action will not complain